core/templates/pages/exploration-player-page/templates/refresher-exploration-confirmation-modal.controller.spec.ts

```diff
@@ -67,7 +70,8 @@ describe('Refresher Exploration Confirmation Modal Controller', function() {
     spyOn(mockWindow, 'open').and.callThrough();
     $scope.confirmRedirect();
 
-    $timeout.flush(150);
+    $flushPendingTasks();
+    $verifyNoPendingTasks('$timeout');
```

Comment:
I think this is good. This removes the variable delay we have. Also, just to make sure, can you run the frontend tests (rerun) on this PR 5 times?

Reply:
@nithusha21 can you please place a review hold so that we merge this only after we have ensured that this does not introduce flakes? Thanks!

Reply:
Hi @nithusha21 actually, I run the script for 5~10 times and it's working perfectly.